bundles/org.openhab.binding.sagercaster/README.md

```diff
@@ -24,9 +24,11 @@ The binding itself does not require any configuration.
 
 | Name               | Type     | Description                                                              |
 |--------------------|----------|--------------------------------------------------------------------------|
-| location           | Location | Latitude and longitude of the desired weather forecast.                  |
+| location (*)       | Location | Latitude and longitude of the desired weather forecast.                  |
 | observation-period | int      | Minimum delay (in hours) before producing forecasts. Defaulted to 6.     |
 
+(*) Only latitude is used by the algorithm.
+
 ## Channels
```

Comment:
Please add the timestamp channel in documentation

Reply:
Done.